src/main/java/uk/nhs/tis/trainee/notifications/event/ProgrammeMembershipListener.java

```diff
@@ -54,7 +57,17 @@ public ProgrammeMembershipListener(ProgrammeMembershipService programmeMembershi
   @SqsListener("${application.queues.programme-membership}")
   public void handleProgrammeMembershipUpdate(ProgrammeMembershipEvent event) {
     log.info("Handling programme membership update event {}.", event);
-    boolean isExcluded = programmeMembershipService.isExcluded(event);
-    log.info("Programme membership {}: excluded {}.", event.tisId(), isExcluded);
+    ProgrammeMembershipMapper mapper = new ProgrammeMembershipMapperImpl();
```

Comment:
This feels like rather more work than the listener should really do, but I didn't want any unexpected-format PM messages (I'm looking at you, COJ_RECEIVED!) to cause issues and be requeued unnecessarily.